components/admin/users/AdminUsersSplitSearch.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+
+/*
+	convert string to array, separated by and including searchTerm
+	Ex:
+		Inputs: searchTerm='bon', str = 'bonJourbon'
+		Output: [bon, Jour, bon]
+*/
+const split = (
+  str: string,
+  lowerCaseSearchTerm: string,
+  searchTerm: string
+) => {
```

Comment:
`const lowerCaseSearchTerm = searchTerm.toLowerCase()` you don't need to pass this in as a variable.